src/settings/BatchGroupConfigurationSettings/BatchVoucherExportsList/BatchVoucherExportsList.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  IconButton,
+  Layout,
+  MultiColumnList,
+} from '@folio/stripes/components';
+import { FolioFormattedTime } from '@folio/stripes-acq-components';
+
+import { BATCH_VOUCHER_EXPORT_STATUS_LABEL } from '../../../common/constants';
+
+const columnMapping = {
+  date: <FormattedMessage id="ui-invoice.settings.BatchVoucherExports.date" />,
+  status: <FormattedMessage id="ui-invoice.settings.BatchVoucherExports.status" />,
+  exportButton: ' ',
+};
+const visibleColumns = ['date', 'status', 'exportButton'];
+const cellFormatters = {
+  date: d => (
+    <FolioFormattedTime dateString={d.end || d.start} />
+  ),
+  exportButton: () => (
+    <Layout element="span" className="full textRight">
```

Comment:
it was a try to align download icon to the right, however it looks like widths of columns should be provided